components/aws/sagemaker/tests/integration_tests/component_tests/test_train_component.py

```diff
@@ -15,6 +15,7 @@
         ),
         pytest.param("resources/config/fsx-mnist-training", marks=pytest.mark.fsx_test),
         "resources/config/spot-sample-pipeline-training",
+        "resources/config/assume-role-training",
```

Comment:
skip the test if assume_role_arn not specified

Reply:
This would make it an optional test? I think it's a necessary integration test?

Reply:
no role_arn is gonna test that the default case works correctly, right

Reply:
Sure, but it will still be provided to the `pytest` which is what I think @surajkota meant? All other tests use the default value.

Reply:
ok, i see you added this as required arg